src/main/scala/com/codacy/client/bitbucket/util/UrlHelper.scala

```diff
@@ -0,0 +1,18 @@
+package com.codacy.client.bitbucket.util
+
+object UrlHelper {
+
+  def joinQueryParameters(baseUrl: String, queryParameters: String*): String = {
```

Comment:
Maybe the receiving the String* is worse than Seq because we need to fold on each one. Sorry for the bad idea..